### PR TITLE
do not strip links and blackquotes

### PR DESCRIPTION
**Pull request by ki88**

### CODE REVIEW

File: test/remove-markdown.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ describe('remove Markdown', function () {
         });
     });
 
-    it('should not remove greater than signs', function () {
+    it.skip('should not remove greater than signs', function () {
```
Comment (alj1s) [inline test/remove-markdown.js]: shouldn't this test still pass?

Comment (ki88) [reply test/remove-markdown.js]: it will not pass because of `{ string: '> 100 > 0', expected: '100 > 0' }`

Comment (alj1s) [reply test/remove-markdown.js]: Can you change the test to check the string `100 > 0`

Comment (ki88) [reply test/remove-markdown.js]: done

**Review by alj1s — APPROVED** [review]

**Pull request merged**